docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+can yield multiple different snippets of the same document.
+
+The semantic search API can also be used based on a specific snippet (and for systems
+which have documents with multiple embeddings that is recommended):
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "effaffe", "sub_id": 2 },
+        "include_properties": true
+    }'
+```
+
+This will return the snippets of which the semantic representation is closest to the
+semantic representation of the second snippet of the given document.
+
+```{note}
+Currently the whole document used for search is excluded from the search results.
+```
+
+For personalize semantic search with provided history snippet ids can also be used instead of
+document ids:
+
+```json
+"personalize": {
+    "exclude_seen": true,
+    "user": {
+         "history": [
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 0 },
+                 "timestamp": "2000-05-14T20:22:50Z"
+             },
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 1 },
+                 "timestamp": "2000-05-15T20:22:50Z"
+             }
+         ]
+    }
+}
+```
+
+When the history contains a interaction with a document which has multiple snippets it will be
+processed similar to registering an interaction which each snippet of that document.
+
+`exclude_seen` currently always refer to whole documents, i.e. in the example above even if there
+is a snippet with the id `{ "document_id": "valid_doc_id1", "sub_id": 30 }` it would exclud a the
+whole document `"valid_doc_id1"` no matter how many snippets it has.
+
+## Personalization
+
+It is possible to register interactions with specific snippets instead of whole documents:
+
+```bash
+curl -X PATCH "$URL/users/u1234/interactions" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "documents": [
+            { "id": { "document_id": "effaffe", "sub_id": 1 } },
+            { "id": { "document_id": "effaffe", "sub_id": 2 } },
+            { "id": { "document_id": "affeffa", "sub_id": 1 } }
+        ]
+    }'
+```
+
+When registering a interaction with a document which has multiple snippets it will be
+processed similar to registering an interaction which each snippet of that document.
+
+# Document preprocessing
+
+## Summarizer
+
+The back office ingestion API provides functionality to automatically summarize the provided content.
+
+This can be used if the content of a document is to large to be used directly as a snippet. Due to the fact that
+the summarized text still has to fit into the size constraints of a snippet it is often better to
+use the [automatic document splitting instead](#automatic-document-splitting).
```

Comment:
```suggestion
This can be used if the content of a document is too large to be used directly as a snippet. Due to the fact that
the summarized text still has to fit into the size constraints of a snippet it is better to
use the [automatic document splitting instead](#automatic-document-splitting).
```